src/tests/Microsoft.PowerFx.Connectors.Tests/Helpers/LoggingTestServer.cs

```diff
@@ -61,6 +61,16 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 _log.AppendLine($" {headerName}: {value}");
             }
 
+            var httpContent = request?.Content;
+            if (httpContent != null)
+            {
+                var content = await httpContent.ReadAsStringAsync();
```

Comment:
>httpContent [](http://example.com/codeflow?start=36&length=11)

Can we log content headers too? (especially content-type)

Reply:
Done